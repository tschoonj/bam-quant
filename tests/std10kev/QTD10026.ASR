File: C:\xrfdaten\2014KW30\chris\std10kev\std10026.spe
Start: 21.07.2014 16:00:49
Stop: 21.07.2014 16:02:29
BESSY: 298.0761705 298.3360212
IONI: 1.444178E-10 1.440284E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 75191
Run: 100000
Dead: 25
ICR: 85630
OCR: 0
Accumulated BESSY: 59681.8755459
Accumulated Ioni: 2.88734488E-08
N Accumulation: 200
Normfactor:       92122449.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
exp_x 152.020401000977 152.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  26   1      6.399   5557356.      2358.     139.76
