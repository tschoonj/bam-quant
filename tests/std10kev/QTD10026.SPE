$DATA:
0 2000
17445
11428
5049
1519
336
44
13
5
7
6
2
2
3
2
1
7
141
1218
1713
1560
1568
1508
1417
1374
1317
1337
1351
1271
1233
1212
1193
1113
1145
1092
1087
1043
1029
984
953
954
965
927
940
907
884
936
919
842
883
892
803
865
795
793
826
777
838
807
788
824
827
829
807
819
721
748
764
775
766
708
806
738
750
770
745
692
728
753
731
694
695
684
693
734
656
701
698
699
706
698
670
751
690
764
684
670
701
723
745
748
768
769
848
782
752
729
700
710
725
670
695
647
689
666
681
674
613
653
678
668
691
663
652
665
671
661
661
699
717
763
753
738
708
639
691
612
657
628
661
657
636
730
757
900
1025
1137
1150
1153
1103
918
829
671
724
680
638
694
666
724
681
750
704
662
643
667
673
619
659
647
672
639
596
653
655
663
649
625
640
622
618
658
647
654
673
635
663
660
693
682
668
674
632
703
679
673
646
701
693
699
705
715
699
725
694
721
716
702
662
705
739
709
722
711
708
742
725
732
771
760
768
803
817
831
886
950
1088
1336
1691
2235
2714
3124
3506
3321
3008
2448
1849
1468
1146
1001
863
899
870
858
920
909
914
924
916
906
983
979
918
949
981
976
1041
988
1099
1115
1199
1229
1262
1345
1415
1426
1414
1428
1452
1331
1370
1345
1327
1449
1398
1374
1352
1372
1370
1404
1353
1407
1408
1454
1338
1407
1385
1443
1460
1555
1501
1725
1772
1823
1853
1897
1983
2118
2249
2407
2459
2800
3026
3158
3604
4066
4485
5455
7212
10929
18937
36319
70107
128915
219879
341097
480648
609308
695205
714541
659366
547634
408172
275954
167280
91702
46487
21508
9899
4918
2853
2129
1903
1806
1781
1917
1890
2010
2329
2539
3004
3867
5144
7506
11572
18885
29702
44447
61854
78851
92114
98971
95076
83488
66904
48264
32371
19255
10737
5544
2675
1233
551
288
167
111
98
112
75
75
73
71
77
53
62
64
60
51
59
47
50
47
42
46
38
40
35
33
31
43
52
38
47
46
51
44
40
29
36
36
38
26
41
24
22
29
23
30
22
26
23
25
23
34
30
31
15
18
19
18
23
19
23
16
19
25
22
23
31
23
23
19
28
26
32
23
20
13
15
18
19
14
28
19
15
18
26
18
15
16
13
20
12
11
11
12
11
18
25
22
18
23
29
19
25
26
24
22
22
31
23
21
25
32
20
25
23
30
46
41
38
41
34
57
65
80
93
117
140
166
199
243
260
347
398
446
390
402
397
365
309
249
194
136
103
77
43
36
27
21
14
9
17
13
15
11
7
8
12
10
12
5
12
12
12
11
10
9
19
10
13
15
18
8
15
13
11
8
17
19
11
19
24
30
28
29
35
48
32
37
39
31
35
32
26
21
22
17
20
16
16
12
19
9
14
17
16
16
12
10
14
8
18
19
17
23
22
13
16
32
20
16
21
26
23
16
21
13
19
42
20
15
13
25
21
14
23
19
13
18
24
20
23
21
12
20
25
29
19
22
31
28
31
32
24
32
36
43
39
44
56
72
78
142
188
298
409
629
1043
1351
1879
2417
2781
3363
3608
3718
3606
3347
2775
2307
1712
1287
883
571
353
253
131
91
52
48
36
42
36
42
48
75
78
124
136
199
279
383
496
634
745
898
952
1066
1019
957
877
753
599
438
307
210
142
82
48
36
18
8
7
4
8
10
6
4
2
5
14
19
23
25
38
41
62
67
71
65
91
68
64
55
46
35
22
18
16
8
6
5
4
3
1
1
0
0
0
1
0
2
0
1
0
1
1
1
1
0
0
0
0
1
0
0
0
1
0
0
1
1
0
0
0
0
1
0
0
0
0
0
0
0
2
0
1
0
0
1
1
0
0
0
0
0
0
0
1
0
0
0
1
1
0
0
0
1
0
0
0
1
0
0
0
1
0
1
2
0
1
0
1
0
0
1
0
0
0
2
2
0
1
0
4
1
2
2
3
6
8
4
3
2
6
6
3
4
4
1
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
2
1
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
1
0
1
0
1
0
0
1
0
1
0
0
0
1
0
0
0
0
0
2
0
0
0
0
1
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
2
0
1
0
1
2
1
0
3
3
1
3
2
4
8
8
6
10
8
11
15
20
9
14
11
10
9
3
3
4
0
2
0
1
0
1
1
0
0
0
0
1
1
0
0
2
2
2
4
3
2
3
2
7
5
10
3
4
5
2
5
3
5
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
2
2
2
1
0
1
0
1
2
1
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10026.spe
Start: 21.07.2014 16:00:49
Stop: 21.07.2014 16:02:29
BESSY: 298.0761705 298.3360212
IONI: 1.444178E-10 1.440284E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 75191
Run: 100000
Dead: 25
ICR: 85630
OCR: 0
Accumulated BESSY: 59681.8755459
Accumulated Ioni: 2.88734488E-08
N Accumulation: 200
Normfactor:       92122449.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Fe
 
exp_x 152.020401000977 152.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  26   1      6.399   5557356.      2358.     139.76
