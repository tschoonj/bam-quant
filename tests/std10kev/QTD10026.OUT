     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:01  บ   
     บ  Spectrum: QTD10026.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   348.4   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -94.3 ๑   100.0   บ     -94.3 ๑      .8   บ   
     บ  GAIN (eV/ch)  บ     20.437 ๑   2.000   บ    20.437 ๑    .003   บ   
     บ det NOISE (eV) บ     -118.9 ๑    40.0   บ    -118.9 ๑    -2.7   บ   
     บ  FANO factor   บ       .271 ๑    .050   บ      .271 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Fe-Ka                           5557356. ๑   2358.           บ   
     บ    KA1         6.399   1.00000     5557356. ๑   2359.    139.8  บ   
     บ              317.721    151.25        2620.           9.17E-01  บ   
     บ                                                                 บ   
     บ  2 Fe-Kb                            814152. ๑    904.           บ   
     บ    KB1         7.059   1.00000      814152. ๑    904.     90.5  บ   
     บ              350.015    163.38        1513.           9.38E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
